osbs/core.py

```diff
@@ -866,6 +866,24 @@ def restore_resource(self, resource_type, resource):
         check_response(response)
         return response
 
+    def create_config_map(self, config_name, config_data):
+        url = self._build_k8s_url("configmaps/")
+        response = self._post(url, data=json.dumps(config_data))
+        check_response(response)
+        return response
+
+    def get_config_map(self, config_name):
+        url = self._build_k8s_url("configmaps/")
+        response = self._get(url, data=json.dumps(config_name))
+        check_response(response)
+        return response
```

Comment:
I think we want to return something the caller can use without having to know the details of what a v1.ConfigMap looks like. See osbs.build.pod_response.PodResponse for the sort of thing I'm thinking of. I think we want an osbs.build.configmap_response.ConfigMapResponse with a get_*() method that retrieves the `data` value.
http://osbs.readthedocs.io/en/latest/multiarch.html#configmapresponse